source/algorithms.tex

```diff
@@ -6286,31 +7997,49 @@
     Otherwise, \tcode{T} meets both
     the \oldconcept{CopyConstructible} (\tref{cpp17.copyconstructible}) and
     \oldconcept{CopyAssignable} requirements.
-    \begin{note}
-    For the overloads with an \tcode{ExecutionPolicy},
-    there might be a performance cost
-    if the value type of \tcode{ForwardIterator1} does not meet both the
-    \oldconcept{CopyConstructible} and \oldconcept{CopyAssignable} requirements.
-    \end{note}
   \end{itemize}
 \end{itemize}
+\begin{note}
+For the parallel algorithm overloads in namespace \tcode{std},
+there can be a performance cost
+if the value type of \tcode{ForwardIterator1} does not meet both the
+\oldconcept{CopyConstructible} and \oldconcept{CopyAssignable} requirements.
+For the parallel algorithm overloads in namespace \tcode{ranges},
+there can be a performance cost if \tcode{iter_value_t<I>}
+does not model both \libconcept{copy_constructible}
+%FIXME: There exists no such concept, but the note refers to it. LWG input needed.
```

Comment:
@jwakely: Note the potential issue.

Reply:
https://lists.isocpp.org/lib/2025/07/32289.php I've asked here now.

Reply:
Thanks!